docs/egress-service.md

```diff
@@ -75,6 +80,40 @@ Assuming an Egress Service has `172.19.0.100` as its ingress IP and `ovn-worker`
 ```
 Notice how the packet exits `ovn-worker`'s eth1 and not breth0, as the packet goes through the host's routing table regardless of the gateway mode.
 
+When `sourceIPBy: "Network"` is set, `ovnkube-master` does not need to create any logical router policies because the egress packets of each pod would exit through the pod's node but will set the status field of the resource with `host: ALL` as decribed later.
+
+### Network
+The `EgressService` supports a `network` field to specify to which network the egress traffic of the service should be steered to.
+When it is specified the relevant `ovnkube-nodes` take care of creating ip rules on their host - either the node which matches `Status.Host` or all of the nodes when `Status.Host` is "ALL".
+Assuming an `EgressService` has `Network: blue`, a ClusterIP of 10.96.135.5 and its endpoints are 10.244.0.3 and 10.244.1.6 the following will be created on the host:
+
+```none
+$ ip rule list
+5000:	from 10.96.135.5 lookup blue
+5000:	from 10.244.0.3 lookup blue
+5000:	from 10.244.1.6 lookup blue
+```
+
+This makes the egress traffic of endpoints of an EgressService to be routed via the "blue" routing table.
+An ip rule is also created for the ClusterIP of the service which is needed in order for the ingress reply traffic (reply to an external client calling the service) to use the correct table - this is because the packet flow of contacting a LoadBalancer service goes:
+`lb ip -> node -> enter ovn with ClusterIP -> exit ovn with ClusterIP -> exit node with lb ip`
+so we need to make sure that packets from ClusterIPs are marked before being routed in order for them to hit the relevant ip rule in time.
+
+### Network without LoadBalancer SNAT
+As mentioned earlier, it is possible to use the "Network" capability without SNATing the traffic to the service's ingress IP. This is done by creating an EgressService with the `Network` field specified and `sourceIPBy: "Network"`.
+
+An EgressService with `sourceIPBy: "Network"` does not need to have a host selected, as the traffic will exit each node with the IP of the interface corresponding to the "Network" by leveraging the masquerade rules that are already in place.
+
+This works only on clusters running on "Local" gateway mode, because on "Shared" gateway mode the ip rules created by the controller are ignored (like all of the node's routing stack).
```

Comment:
is there a check in your current code to detect this unsupported case and log an error or post an event to the egress service crd?

Reply:
added a log in the clustermanager